README.md

```diff
@@ -22,3 +22,7 @@ The vault takes heavy inspiration from the following sources:
 - [Popcorn DAO - MultiRewardStaking](https://github.com/Popcorn-Limited/contracts/blob/d029c413239735f58b0adcead11fdbe8f69a0e34/src/utils/MultiRewardStaking.sol)
 
 This is a simplified implementation that keeps only the core functionality of the vaults and removes any additional features.
+
+## WARNING
+
+** When deploying GenericStakedAppreciatingVault, make sure to mint some initial dust (at least 1e4), to avoid an inflation attack. **
```

Comment:
Clarify that this is only required for large assets where manipulation is profitable with a `0` decimal offset. The vault internally already uses virtual assets and shares making it not profitable for most assets even with a `0` decimal offset.

(Also, nit, remove the excess bolding and capitalization and use code markers for contract names and symbols)